crates/re_space_view/src/space_view_contents.rs

```diff
@@ -270,7 +304,14 @@ impl DataBlueprintTree {
         let mut new_leaf_groups = Vec::new();
 
         for path in paths {
-            self.entity_paths.insert(path.clone());
+            // TODO(andreas/jleibs): Incoming entities should already "know" what system they belong to.
```

Comment:
This doesn't need to happen at all -- we're going to recompute it based on the group hierarchy on every frame at which point it won't be unknown.